modules/cpmso-ts-ilb-missing.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="cpmso-ts-ilb-missing_{context}"]
+= Adding a missing Azure internal load balancer
+
+The `internalLoadBalancer` parameter is required in both the `ControlPlaneMachineSet` and control plane `Machine` custom resources (CRs) for Azure. If this parameter is not preconfigured on your cluster, you must add it to both CRs.
+
+For more information on where this parameter is located in the Azure provider specification, see the sample Azure provider specification. The placement in the control plane `Machine` CR is similar.
+
+.Procedure
+
+//Do they actually have to do this or do they just need to stick it in the CPMS CR and it will propagate from there?
```

Comment:
If they don't they'll create a diff between the Azure machines and CPMS and that will cause a rollout